### ISSUE TITLE
Ballot box in moderator election is hidden

### ISSUE THREAD

**Issue by ooker777** [soscripted/sox#502]
https://literature.stackexchange.com/election

Without SOX:
![image](https://user-images.githubusercontent.com/6695517/230062730-9182dfd0-45a1-4638-b107-5c0b43316db1.png)

With SOX:
![image](https://user-images.githubusercontent.com/6695517/230062813-83b9e26d-a3ec-4666-837c-54950643b037.png)

I'm on Firefox 111.0.1, SOX 2.8.4dev on Tampermonkey

**Comment by shu8** [comment]
Thanks for the bug report. Unfortunately, I don't have 150 rep on Literature so can't try this myself. Do you know if there are any other sites with an active election?

In any case, please could you paste your current enabled settings here so I can try finding which feature is causing it (I suspect it will be one of the 'sidebar'-related features). This list should have been auto-populated in your GH issue when you started creating it.

**Comment by ooker777** [comment]
Unfortunately there is no site which I'm participating that currently having election. The setting wasn't uto-populated when I started the issue. Here it is:

```
["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-localTimestamps","Appearance-metaChatBlogStackExchangeButton","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-showTagWikiLinkOnTagPopup","Appearance-hideWelcomeBackMessage","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-sortByBountyAmount","Extras-copyCode","Extras-openLinksInNewTab","Extras-addOnTopicLinkToSiteSwitcher","Extras-customMagicLinks"]
```

**Comment by shu8** [comment]
So this was being caused by the `Sidebar-hideCommunityBulletin` ("Hide the Community Bulletin module") feature. You can confirm it on https://literature.stackexchange.com/election/2 -- if you disable that feature you will see something like this in the sidebar: 

<img width="160" alt="image" src="https://github.com/soscripted/sox/assets/8850830/dddb2ea2-8e99-4125-aaf0-5d8927227803">

but with it enabled, it won't be there.

It should now be fixed in [dev v2.8.5](https://github.com/soscripted/sox/raw/dev/sox.user.js)!

(for future reference for myself, https://meta.stackexchange.com/questions/287521/what-current-moderator-elections-are-happening is useful!)
